src/Psy/Configuration.php

```diff
@@ -721,6 +721,14 @@ public function setTabCompletion($tabCompletion)
     }
 
     /**
+     * @param array $matchers
+     */
+    public function setTabCompletionMatchers(array $matchers)
+    {
+        $this->tabCompletionMatchers = array_merge($this->tabCompletionMatchers, $matchers);
+    }
```

Comment:
This is now `addTabCompletionMatchers` for consistency with existing things, and consistency with what the words mean :)